docs/getting-started/styling-messages/markdown.mdx

```diff
@@ -0,0 +1,284 @@
+    <tr>
+        <td>__underlined__</td>
+        <td>~~strikethrough~~</td>
+    </tr>
+</table>
+
+## Code Markdown
+
+This formatting can be used not only for code but also for normal text,
+when you want the text to have a different background or to be encased in a box.
+
+### Single-Line Code Block
+
+To create a single-line code block without syntax highlight, wrap the text between single backticks.
+This code block will only add a dark background to the text.
+
+```text
+`print("Hello world!")`
+```
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+`print("Hello world!")`
+    </DiscordMarkdown>
+  </DiscordMessage>
+</DiscordComponent>
+
+<br/>
+
+### Multi-Line Code Block
+
+This type of code block, created by using triple backticks, will encase the text inside a box.
+
+#### Without Syntax Highlight
+
+To create a multi-line code block without syntax highlight, wrap the text between triple backticks
+on first and last line.
+
+````text
+```
+message = "Hello world!"
+print(message)
+```
+````
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+```
+message = "Hello world!"
+print(message)
+```
+    </DiscordMarkdown>
+  </DiscordMessage>
+</DiscordComponent>
+
+<br/>
+
+#### With Syntax Highlight
+
+To create a multi-line block with syntax highlight, add the name of the language you are using right after
+the triple backticks on the first line. For example, for Python you can write either "python" or "py".
+
+````
+```python
+message = "Hello world!"
+print(message)
+```
+````
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+```
+message = "Hello world!"
+print(message)
+```
+    </DiscordMarkdown>
+  </DiscordMessage>
+</DiscordComponent>
+
+<br/>
+
+### Single-Line Code Block
+
+If you want to use syntax highlight for a single line of code, you still have to format it
+like a multi-line block, however, you cannot start writing the encased text on the same line
+where you declare which language to use.
+
+````
+```python
+print("Hello world!")
+```
+````
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+```
+print("Hello world!")
+```
+    </DiscordMarkdown>
+  </DiscordMessage>
+</DiscordComponent>
+
+<br/>
+
+## Quote Markdown
+
+### Single-Line Quote Block
+
+To create a single-line quote block, start the line with `>` followed by a space.
+
+```text
+> This is a single-line quote.
+```
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+> This is a single-line quote.
+    </DiscordMarkdown>
+  </DiscordMessage>
+</DiscordComponent>
+
+<br/>
+
+### Multi-Line Quote Block
+
+To create a multi-line quote block, write `>>>` followed by a space. All text
+from that point onwards will be included in that quote block.
+
+```text
+>>> This is a
+multi-line quote.
+```
+
+<DiscordComponent>
+  <DiscordMessage profile="robocord">
+    <DiscordMarkdown>
+> This is a
```

Comment:
If we write `>>>`, we get an error. If we write `>` for each line, the first line shows as quote block correctly but the second `>` shows as text instead of being rendered as quote block.